cmd/blockchaincmd/add_validator.go

```diff
@@ -186,13 +179,7 @@ func addValidator(cmd *cobra.Command, args []string) error {
 	}
 
 	if len(args) == 0 {
-		if rpcURL == "" {
-			rpcURL, err = app.Prompt.CaptureURL("What is the RPC endpoint?", false)
```

Comment:
automatic prompting associated with flags is a very very nice idea, but we need to consider that depending on 
context, flags will or wont be required. and the context is maybe another flags or another prompts.

Reply:
I totally agree with automatic default initialization, validation, and prompting if missing, but we need input values dependences and relationships into the solution